modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -46,6 +47,14 @@ function widget_admin_setup() {
 		wp_enqueue_script( 'widget-jetpack-search-filters' );
 	}
 
+	private function get_sort_types() {
+		return array(
+			'relevance|DESC' => esc_html( 'Relevance' ),
```

Comment:
As written, this string is not translated.

`esc_html( 'Relevance' )` => `esc_html__( 'Relevance', 'jetpack' )`

Same with the next two strings.